### PR TITLE
Stream query

### PR DESCRIPTION
**Pull request by h-2**
blocked by #154 

(only the last commit is relevant is specific to this PR)

Includes progress bar again, although I haven't checked how fast counting is; likely room for optimisation.

### CODE REVIEW

**Comment by sarahet** [comment]
All builds fail ..

File: src/search_algo.hpp [hunk 1]
```diff
@@ -1215,10 +1125,10 @@ iterateMatchesFullSerial(TLocalHolder & lH)
 
     auto const trueQryId = lH.matches[0].qryId / qNumFrames(TGlobalHolder::blastProgram);
 
-    TBlastRecord record(lH.gH.qryIds[trueQryId]);
+    TBlastRecord record(lH.qryIds[trueQryId]);
     record.qLength = seqan::length(lH.gH.qrySeqs[trueQryId]);
```
Comment (sarahet) [inline src/search_algo.hpp]: This needs to be ```record.qLength = seqan::length(lH.qrySeqs[trueQryId]);```

Comment (h-2) [reply src/search_algo.hpp]: Yes, that's artifact from merging the branches.

**Comment by sarahet** [comment]
Since we talked about divergence of branches, should be maybe address this here as well and account for the recent changes in the master branch?

**Comment by h-2** [comment]
> Since we talked about divergence of branches, should be maybe address this here as well and account for the recent changes in the master branch?

No, we should merge master into lambda3 at some point and resolve the conflicts. But it doesn't have to happen in the next days.

**Pull request merged**